community-plugins.json

```diff
@@ -4832,4 +4832,12 @@
         "repo": "czottmann/obsidian-copy-search-url",
         "branch": "main"
     }
```

Comment:
There is a comma missing here, blocking our validation script from running.
This would also block Obsidian itself from showing any plugins if this were merged.